docs/scan-sequence.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+autonumber
+
+actor       User       as user
+participant "Workload controller" as wc
+entity      Pod      as pod
+participant "CIS controller" as cc
+entity      CIS      as cis
+participant "Scan Job controller" as jc
+entity      Job      as job
+
+user -> pod : create
+activate pod 
+
+wc <<- pod : watch
+activate wc
+wc -> cis : create
+deactivate wc
+activate cis
+
+cc <<- cis : watch
+activate cc
+cc -> job : create
+deactivate cc
+activate job
+
+|||
+
+jc <<- job: watch
+deactivate job
+activate jc
+jc -> cis : update status
+deactivate jc
+
```

Comment:
Maybe also add what happens when the pod is deleted?
```suggestion
user -> pod : delete
pod -->> cis : delete (GC)
deactivate pod 
deactivate cis
```
![image](https://user-images.githubusercontent.com/12656918/214782994-c703136a-f4fc-4e17-8f0c-ed37cf08e918.png)

Reply:
Good suggestion! Updated, PTAL!